components/ai_chat/core/browser/engine/oai_api_client.cc

```diff
@@ -150,7 +150,11 @@ void OAIAPIClient::OnQueryCompleted(GenerationCompletedCallback callback,
           DVLOG(2) << "No message dict found in response.";
           return;
         }
-        completion = *message->FindString("content");
+
+        const std::string* content = message->FindString("content");
+        if (content) {
+          completion = *content;
+        }
```

Comment:
Please add something like `PerformRequest_BadResponse` or `PerformRequest_InvalidResponse` test case in oai_api_client_unittest and cover these checks.